### ISSUE TITLE
APA site detection beyond the 3'-UTR?

### ISSUE THREAD

**Issue by lennartraman** [arefeen/TAPAS#19]
Hi devs,

First of all, thanks for TAPAS!

My question is quiet simple: can TAPAS only detect APA sites within the 3'-UTR, or can it also detect APAs more upstream (intronic/exonic)?

Thanks for your valuable insight.

Best Lennart

**Comment by arefeen** [comment]
Hi Lennart,

I am glad that you like TAPAS.

It can detect APA sites in 3'-UTR region. If you want to detect APA sites
in intronic or exonic region please use DeepPASTA (another APA site
detection tool from our lab).

Thanks,
Ashraful

**Comment by lennartraman** [comment]
Hi @arefeen 

Thanks for your quick response! Just to be sure: DeepPASTA can detect APAs in, let's say, the complete gene, based on mRNA sequencing data as input?

Thanks a lot,
Lennart

**Comment by arefeen** [comment]
Yes, that's right. Please read the webpage of DeepPASTA for more
information.

**Comment by lennartraman** [comment]
Alright, thanks a lot!
